packages/bundle-size/src/commands/uploadReport.js

```diff
@@ -0,0 +1,139 @@
+const { odata, TableClient, TablesSharedKeyCredential, TableTransaction } = require('@azure/data-tables');
+const chalk = require('chalk');
+
+const collectLocalReport = require('../utils/collectLocalReport');
+const { hrToSeconds } = require('../utils/helpers');
+
+/** @typedef {import('../utils/collectLocalReport').BundleSizeReportEntry} BundleSizeReportEntry */
+
+const AZURE_STORAGE_ACCOUNT = 'fluentbundlesize';
```

Comment:
can we move this to some standard configurable format outside from javascript ? json for example

Reply:
https://github.com/microsoft/fluentui/pull/18322#discussion_r657806360